app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -102,9 +102,42 @@ def generate_live_feedback
     response_status, response_body = @answer.generate_live_feedback
     response_body['feedbackUrl'] = ENV.fetch('CODAVERI_URL')
 
+    course_user = CourseUser.find_by(course_id: @assessment.course_id, user_id: @submission.creator_id)
+    live_feedback = Course::Assessment::LiveFeedback.create_with_codes(
+      @submission.assessment_id,
+      @answer.question_id,
+      course_user.id,
+      response_body['transactionId'],
+      @answer.actable.files
+    )
+
+    if response_status == 200
```

Comment:
Because when response status is 200, live feedback is immediately available to be saved. FE wont call save_live_feedback. The live feedback is sent directly to FE from BE and frontend does no polling

Reply:
But when I try to generate the live feedback in the frontend (by clicking Get Help), there will be a network call for `save_live_feedback`.

If when the response status is 200 the live feedback is immediately available, I think there should not be another call of save_live_feedback from FE; it should be silently saved in BE.

Reply:
@bivanalhar this is because when user clicks [Get Help], we generate a LiveFeedback entry without any associated comments first, and only populate the comments once those comments returned by Codaveri. 

Nevertheless, we increment the count since user has indeed requested [Get Help].